package.json

```diff
@@ -186,6 +186,7 @@
     "moment-duration-format": "^2.2.2",
     "mz": "^2.7.0",
     "node-cleanup": "^2.1.2",
+    "nyc": "^14.1.1",
```

Comment:
Should this be under "devDependencies" instead of "dependencies"? (In which case, `npm-shrinkwrap.json` would also need to be regenerated.)

Reply:
Ah yes, good catch @pdcastro